src/main/java/com/mineinabyss/bonfire/listeners/BlockListener.kt

```diff
@@ -95,12 +96,12 @@ object BlockListener : Listener {
 
         transaction {
             val hasRegisteredPlayers = Players.select { Players.bonfireUUID eq bonfire.uuid }.any()
+            val bonfireRow = Bonfire.select { Bonfire.entityUUID eq bonfire.uuid }.first()
```

Comment:
There is a theoretical possibility (though it shouldn't happen) that the bonfire lacks a corresponding database entry. I always like to make sure that the plugin doesn't throw exceptions in that case, so you could do a `firstOrNull()` call. In this case that would mean that the `else { }` block below gets called, which is perfect.

Reply:
This should be fixed now.

Reply:
I would give this one last test, to ensure this is working.